src/Nethermind/Nethermind.Blockchain.Test/Receipts/PersistentReceiptStorageTests.cs

```diff
@@ -68,5 +70,66 @@ public void Get_returns_empty_on_empty_span()
         {
             _storage.Get(Keccak.Zero).Should().BeEquivalentTo(Array.Empty<TxReceipt>());
         }
+        
+        [Test]
+        public void Adds_and_retrieves_receipts_for_block()
+        {
```

Comment:
can we have a test for:
Get -> empty
Get (risk that from cache) -> empty
Insert
Get (should not be empty?)